wordfence/wordpress/site.py

```diff
@@ -522,5 +522,6 @@ def get_database(self) -> WordpressDatabase:
         return WordpressDatabase(
```

Comment:
Does the
```
try:
    prefix = config['prefix']
except KeyError:
    prefix = DEFAULT_PREFIX
```
block (used in `dbscan.py`) want to be here too? Both collation and prefix are in `dbscan.py` so not sure if it's necessary or desirable?